src/react/ui-elements/ListLayoutButtons.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useMemo } from 'react';
```

Comment:
This `Buttons` Redux container is not really an ui component and is very specific to data browser, so we should move it to `databrowser` directory.